decidim-core/spec/mailers/reported_mailer_spec.rb

```diff
@@ -4,6 +4,8 @@
 
 module Decidim
   describe ReportedMailer, type: :mailer do
+    include Decidim::SanitizeHelper
```

Comment:
This `include` is not necessary. I checked it locally and it ran without any failure.